libctru/include/3ds/services/ac.h

```diff
@@ -10,11 +10,11 @@ Result acInit(void);
 /// Exits AC.
 void acExit(void);
 
+/// Waits for the system to connect to the internet.
+Result acWaitInternetConnection(void);
```

Comment:
It actually isn't an IPC command, for some reason it was uppercased. This is what it was before: https://github.com/Steveice10/ctrulib/commit/7e5edd91385f5943899de00f4de5e40bc91d3c9c#diff-8ff965f94c5b943d375ce717582bc9c7L93